ex8/js/config.js

```diff
@@ -0,0 +1,7 @@
+requirejs.config({
+  baseUrl: 'js',
+  paths: {
+    ajax: 'ajax',
```

Comment:
You should use requirejs' `paths` for other people's code, not yours. 
This just alias `updatePhoneBook` to `update`. If you want it to be called `update`, rename the file; if you want it to be called `updatePhoneBook`, change the require call for it.